Cart/cart.css

```diff
@@ -70,14 +70,14 @@
 }
 
 .checkout-container {
-    border: 1px solid black;
+    border: 1px solid var(--prime-blue);
     padding: 2rem;
     width: 400px;
```

Comment:
You can `rem` here.

Reply:
For smaller units like 1px, pixel unit is fine

Reply:
Thanks for pointing it out, will make the change